### PR TITLE
Portfolio Mobile menu

### PR DESCRIPTION
**Pull request by brahimdidi**
In this project, we will create a mobile menu using Javascript and by manipulating the dom.
## Requirements
When the user clicks (or taps) the hamburger button, the mobile menu appears.
When the user clicks (or taps) the close (X) button, the mobile menu disappears.
When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.
There are no linters errors

### CODE REVIEW

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi , 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚥 🚦  

## Good Points 👍  
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._
- [ ] Your project files are overly huge. Countless attempts to clone this project have proven futile. Kindly streamline your project files, and leave out the files considered unnecessary. Kindly confirm that this repository is clonable before resubmitting for approval.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked [OPTIONAL] under the review._
<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+const ulLinks3 = document.querySelector('#ulLinks3');
+// Create the arrows
+const arrowRight = document.createElement('img');
+const arrowRight2 = document.createElement('img');
+const arrowRight3 = document.createElement('img');
+arrowRight.src = 'template5Images/arrowRight.svg';
+arrowRight2.src = 'template5Images/arrowRight.svg';
+arrowRight3.src = 'template5Images/arrowRight.svg';
+
+// crate the cancel button
```
[the remark this answers is not in the document]
Comment (brahimdidi) [reply main.js:18]: you just had nothing to note so you commented on this  :rofl:

File: index.html [hunk 2]
```diff
@@ -28,13 +28,13 @@
...
-			<a href="#section1">Portofolio</a>
-			<a href="#section2">About</a>
-			<a href="#footer">Contact</a>
-			<a href="#"><img src="desktopimages/Union.svg" alt="Icon Index"></a>
+       <nav class="dsk-btn"> 
+			<a href="#section1" id="ulLinks">Portofolio</a>
+			<a href="#section2" id="ulLinks2">About</a>
+			<a href="#footer" id="ulLinks3">Contact</a>
+			<a href="#footer" id="message-icon"><img src="desktopimages/Union.svg" alt="Icon Index"></a>
 		</nav>
```
[the remark this answers is not in the document]
Comment (brahimdidi) [reply index.html:38]: the first time I hear about spaces, Man you really just wanted to make comments.

File: main.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+const cancelbtn = document.createElement('img');
+cancelbtn.src = 'template5Images/Icon - Cancel.svg';
+
+const open = function open() {
```
[the remark this answers is not in the document]
Comment (brahimdidi) [reply main.js:22]: that's the only relevant comment I saw

**Review by AshakaE — APPROVED** [review]
### Hi @brahimdidi  🙋‍♀️, 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bellom — COMMENTED** [review]
Hello 👋,

This is an additional review as a result of the second opinion request.

Looks like the issue was resolved here, so I have nothing else to say other than best of luck with the rest of your projects!

Keep rocking 🚀!

**Pull request merged**